src/controllers/users.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable space-before-function-paren */
+'use strict';
+
+const {
+  getAll,
+  getUserById,
+  create,
+  remove,
+  update,
+  normalize,
+} = require('../services/users');
+
+const getAllUsers = async (req, res) => {
+  const users = await getAll();
+
+  res.send(users.map(user => normalize(user)));
```

Comment:
Here you can pass to the map just `normalized ` function because map gets a callback which it executes for each item.
`users.map(normalize);`